pkg/bindings/containers/archive.go

```diff
@@ -49,12 +49,17 @@ func Stat(ctx context.Context, nameOrID string, path string) (*entities.Containe
 	return statReport, finalErr
 }
 
-func CopyFromArchive(ctx context.Context, nameOrID string, path string, reader io.Reader) (entities.ContainerCopyFunc, error) {
+func CopyFromArchive(ctx context.Context, nameOrID string, path string, reader io.Reader, options *CopyOptions) (entities.ContainerCopyFunc, error) {
```

Comment:
Sorry for noticing just now.  This would break the bindings which is something we cannot do at the moment without having to bump Podman to 4.0.